QualityControl/public/Model.js

```diff
@@ -362,4 +367,58 @@ export default class Model extends Observable {
     this._isUpdateVisible = value ? true : false;
     this.notify();
   }
+
+  /**
+   * Checks if run mode is activated
+   * @returns {boolean} true if activated
+   */
+  get inRunMode() {
+    return this._inRunMode;
+  }
+
+  enterRunMode() {
+    this._inRunMode = true;
+    this._runNumber = this.router.params.RunNumber || null;
+    this.notify();
+  }
+
+  exitRunMode() {
+    this._inRunMode = false;
+    this._runNumber = null;
+    this._runStatus = null;
+    this.notify();
+  }
+
+  /**
+   * Get run number
+   * @returns {null | number} Run number or null if not set
+   */
+  get runNumber() {
+    return this._runNumber;
+  }
+
+  /**
+   * Activate/Deactivate runs mode
+   * @returns {null | string} Run status or null if not set
+   */
+  get runStatus() {
+    return this._runStatus;
+  }
+
+  /**
+   * Sets the current run status and triggers a notification update.
+   * @param {string} value - The new run status (e.g., 'ONGOING', 'COMPLETED').
```

Comment:
Where is `COMPLETED` coming from? As far as I can see from your previous PR status is one of
```
  ENDED: 'ENDED',
  ONGOING: 'ONGOING',
  NOT_FOUND: 'NOT_FOUND',
```